scalardl-node-client-sdk.js

```diff
@@ -27,23 +27,174 @@ class ClientService extends ClientServiceBase {
     let ledgerClient;
     let ledgerPrivilegedClient;
     if (tlsEnabled) {
-      ledgerClient = new LedgerClient(ledgerClientUrl,
-          grpc.credentials.createSsl(Buffer.from(ca, 'utf8')));
-      ledgerPrivilegedClient = new LedgerPrivilegedClient(ledgerClientUrl,
-          grpc.credentials.createSsl(Buffer.from(ca, 'utf8')));
+      ledgerClient = new LedgerClient(
+          ledgerClientUrl,
+          grpc.credentials.createSsl(Buffer.from(ca, 'utf8')),
+      );
+      ledgerPrivilegedClient = new LedgerPrivilegedClient(
+          ledgerClientUrl,
+          grpc.credentials.createSsl(Buffer.from(ca, 'utf8')),
+      );
     } else {
-      ledgerClient = new LedgerClient(ledgerClientUrl,
-          grpc.credentials.createInsecure());
+      ledgerClient = new LedgerClient(
+          ledgerClientUrl,
+          grpc.credentials.createInsecure(),
+      );
       ledgerPrivilegedClient = new LedgerPrivilegedClient(
           ledgerPrivilegedClientUrl,
-          grpc.credentials.createInsecure());
+          grpc.credentials.createInsecure(),
+      );
     }
     const services = {
       'ledgerClient': ledgerClient,
       'ledgerPrivileged': ledgerPrivilegedClient,
     };
     super(services, protobuf, properties);
   }
+
+  /**
+   * @param {Uint8Array} serializedBinary
```

Comment:
```suggestion
   * @param {Uint8Array} serializedBinary a serialized byte array of CertificateRegistrationRequest
```
I'm not sure if we can hav a comment like this in nodejs though.

Reply:
No problem.
It is a correct JSDoc.

Reply:
No problem.
It's a correct JSDoc.